src/h5web/visualizations/shared/VisCanvas.tsx

```diff
@@ -34,7 +34,8 @@ function VisCanvas(props: Props): JSX.Element {
           <Canvas
             className={styles.canvasWrapper}
             orthographic
-            invalidateFrameloop
+            invalidateFrameloop // disable game loop
+            gl={{ preserveDrawingBuffer: true }} // for screenshot feature
```

Comment:
Does this impact performance ?

Reply:
Hehe excellent question -- it doesn't _look_ like it, but it may increase memory usage and impact performance on some devices. I'm hoping they disable it by default because it's a bad thing to enable with a game loop, but since we disable the game loop... It didn't notice a slow down in the interactions, so in my opinion we should be fine. If we ever have memory issues, then we'll know to look here.